tests/models/test_default_evaluator.py

```diff
@@ -1578,3 +1579,30 @@ def test_evaluation_metric_name_configs(prefix, log_metrics_with_dataset_info):
 
     # Dataset info should only be included in logged metric names
     assert all("on_data_iris" not in metric_name for metric_name in result.metrics)
+
+
+@pytest.mark.parametrize("pos_label", [0, 1])
+def test_evaluation_pos_label(pos_label):
+    X, y = load_breast_cancer(as_frame=True, return_X_y=True)
+    X = X.iloc[:, :4].head(100)
+    y = y.head(len(X))
+    with mlflow.start_run():
+        model = LogisticRegression()
+        model.fit(X, y)
+        model_info = mlflow.sklearn.log_model(model, "model")
+        result = evaluate(
+            model_info.model_uri,
+            X.assign(target=y),
+            model_type="classifier",
+            targets="target",
+            dataset_name="breast_cancer",
+            evaluators="default",
+            evaluator_config={"pos_label": pos_label},
+        )
+        y_pred = model.predict(X)
+        precision = precision_score(y, y_pred, pos_label=pos_label)
+        recall = recall_score(y, y_pred, pos_label=pos_label)
+        f1 = f1_score(y, y_pred, pos_label=pos_label)
+        np.testing.assert_almost_equal(result.metrics["precision"], precision)
+        np.testing.assert_almost_equal(result.metrics["recall"], recall)
+        np.testing.assert_almost_equal(result.metrics["f1_score"], f1)
```

Comment:
Can we also add a test case for multi-class classification?

Reply:
Got it, I'll add them.

Reply:
Added.

Reply:
Added.